libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -124,11 +125,17 @@ const AP_Param::GroupInfo AP_Vehicle::var_info[] = {
     AP_SUBGROUPINFO(nmea, "NMEA_", 17, AP_Vehicle, AP_NMEA_Output),
 #endif
 
+#if AP_DDS_ENABLED
+    // @Group: XRCE
+    // @Path: ../AP_DDS/AP_DDS_Client.cpp
+    AP_SUBGROUPPTR(dds_client, "XRCE_", 18, AP_Vehicle, AP_DDS_Client),
+#endif
+
     AP_GROUPEND
 };
 
 // reference to the vehicle. using AP::vehicle() here does not work on clang
-#if APM_BUILD_TYPE(APM_BUILD_UNKNOWN) || APM_BUILD_TYPE(APM_BUILD_AP_Periph)
```

Comment:
why this change? I think this is what is causing the CI error

Reply:
Reverted it in 7d025e82e693825435d9c342b5d776a8fa03f552, I'll watch CI to see what changes.

Reply:
Good eye, I removed it in 7d025e82e693825435d9c342b5d776a8fa03f552. 

You can see the resulting CI pass the build here. 
https://github.com/Ryanf55/ardupilot/actions/runs/4401148084

Resolved :sunglasses: